eventstore/models.py

```diff
@@ -709,33 +699,38 @@ def process_study_c(self, risk, source):
                     self.msisdn,
                     self.hcs_study_c_testing_arm,
                     self.hcs_study_c_quarantine_arm,
-                    None,
                     risk,
                     source,
                 )
 
-    def process_study_c_pilot(self, risk, source):
-        if not settings.HCS_STUDY_C_PILOT_ACTIVE:
-            return
+    def get_study_totals_per_province(self):
+        all_provinces = HealthCheckUserProfile.objects.filter(
+            hcs_study_a_arm__isnull=False
+        ).values("province")
 
-        if not self.hcs_study_c_pilot_arm and risk == Covid19Triage.RISK_HIGH:
-            self.hcs_study_c_pilot_arm = self.get_random_study_pilot_arm()
-            update_turn_contact.delay(
-                self.msisdn, "hcs_study_c_pilot_arm", self.hcs_study_c_pilot_arm
-            )
-            start_study_c_registration_flow.delay(
-                self.msisdn, None, None, self.hcs_study_c_pilot_arm, risk, source
-            )
+        actual_total = {
+            province["province"]: province["province__count"]
+            for province in all_provinces.annotate(models.Count("province"))
+        }.get(self.province, 0)
+
+        actual_percentage = {
+            province["province"]: province["province__count"] * 100 / len(all_provinces)
+            for province in all_provinces.annotate(models.Count("province"))
```

Comment:
I feel like this would run the expensive Count per province twice, also not sure if `len(all_provinces)` will do to count every time.. I'm wondering if that's fine, or if we should only do the count once.

`assertNumQueries` is a good way to check how many and what queries using unit tests.

Also, getting the values for all provinces, and then only returning a single province, seems a bit wasteful. Maybe something like:
```
all_total = all_provinces.count()
province_total = all_provinces.filter(province=self.province).count()
province_percentage = province_total * 100 / all_total
```